### PR TITLE
[mdtool] Call terminate when ending

### PR DESCRIPTION
**Pull request by iainx**
Fixes VSTS #725899

### CODE REVIEW

File: main/src/tools/mdtool/mdtool.csproj [hunk 1]
```diff
@@ -29,6 +29,10 @@
       <Name>Mono.Addins.Setup</Name>
       <Private>False</Private>
     </ProjectReference>
+    <ProjectReference Include="..\..\core\MonoDevelop.Ide\MonoDevelop.Ide.csproj">
+      <Project>{27096E7F-C91C-4AC6-B289-6897A701DF21}</Project>
+      <Name>MonoDevelop.Ide</Name>
+    </ProjectReference>
```
Comment (mrward) [inline main/src/tools/mdtool/mdtool.csproj]: Missing Private false?

Comment (iainx) [reply main/src/tools/mdtool/mdtool.csproj]: Fixed

**Comment by mrward** [comment]
Seem to now include some external repo changes. I guess by accident.

**Comment by iainx** [comment]
@mrward oh how I hate git submodule! Fixed

**Pull request merged**